### ISSUE TITLE
`to_nested_dict` results in error in python 3. Ilastikrag python3 ready?

### ISSUE THREAD

**Issue by k-dominik** [ilastik/ilastikrag#1]
In the ilastik multicut workflow in python 3, an issue arises with the following lines

https://github.com/stuarteberg/ilastikrag/blob/0fa508f308a75ffac9ab0515b4eb3e05b6ff1653/ilastikrag/gui/util/hierarchical_checklist_view.py#L182-L184

which result in 

```python traceback
RuntimeError: OrderedDict mutated during iteration
```
.

As a workaround one could populate a list of values that should be deleted:
```python
to_delete  = [name for name in d.keys()
              if (isinstance(d[name], dict) and len(d[name]) == 0)]
for name in to_delete:
    del d[name]
```

But the question here would be, if ilastikrag is python3 ready at all?

**Comment by k-dominik** [comment]
Ah. @akreshuk fixed that issue in the ilastik fork 17 days ago already...

**Comment by stuarteberg** [comment]
The ilastik build should use the ilastik org fork of this repo:

https://github.com/ilastik/ilastikrag

In that fork, Anna already fixed this issue:
https://github.com/ilastik/ilastikrag/commit/addf55bab031af910aadddeb28c94206cfaa1932

**Comment by stuarteberg** [comment]
😆 I'm not fast enough!

**Comment by k-dominik** [comment]
You are super fast, i should have checked more thoroughly

**Comment by stuarteberg** [comment]
Fixed via addf55b, merged from the ilastik fork.
